.github/workflows/main.yml

```diff
@@ -90,116 +90,81 @@ jobs:
         path: OctoBot/dist/OctoBot_windows.exe
         if-no-files-found: error
 
-  create-release:
```

Comment:
I think we should still publish a release add provide a download link to the created binaries from the release desc

Reply:
Good idea 👍